### ISSUE TITLE
Error UNKNOWN_EXCEPTION: Unable to handle Aura Definition 'OrderedList.cmp': filepath not expected

### ISSUE THREAD

**Issue by anshsetia1818** [rapsacnz/DualSelect#1]
While deploying to Org. it throws error.
Error UNKNOWN_EXCEPTION: Unable to handle Aura Definition 'OrderedList.cmp': filepath not expected

**Comment by rapsacnz** [comment]
I'm having trouble figuring this one out. For now, just manually copy the components into your org - if you find any errors on save, please let me know, as they are likely the same issue that is preventing this deployment.

**Comment by rapsacnz** [comment]
I manually created the component - there were no issues and all works well. I have to assume something is wrong with the deploy to salesforce button in this instance. If I get time, I may create my own package.xml file to try to alleviate this issue.

**Comment by anshsetia1818** [comment]
Yes you are right by manually copy pasting its working perfectly fine.
Just 1 bug i found when i select any value from left n click on arrow
button to move right it will move for first time.
Now if without selecting any further value user clicks same button n number
of times same value get added n number of times to right.

On Sep 23, 2017 7:02 AM, "Caspar Harmer" <notifications@github.com> wrote:

I manually created the component - there were no issues and all works well.
I have to assume something is wrong with the deploy to salesforce button in
this instance. If I get time, I may create my own package.xml file to try
to alleviate this issue.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/rapsacnz/DualSelect/issues/1#issuecomment-331598419>,
or mute the thread
